composer.json

```diff
@@ -1,7 +1,6 @@
 {
 	"require": {
 		"symfony/finder": "~2.3",
-		"symfony/filesystem": "~2.3",
-		"wp-cli/php-cli-tools": "0.10.3"
```

Comment:
Because our `wp-launch-check.phar` requires WP-CLI, it's no longer necessary to bundle our own copy of `php-cli-tools`